src/pages/Fallback/Fallback.js

```diff
@@ -6,15 +6,15 @@ import { FlexBoxAlignItems } from '@ui5/webcomponents-react/lib/FlexBoxAlignItem
 import { FlexBoxDirection } from '@ui5/webcomponents-react/lib/FlexBoxDirection';
 import { FlexBoxJustifyContent } from '@ui5/webcomponents-react/lib/FlexBoxJustifyContent';
 
-import BrowserURL from '../../util/BrowserURL'
+import BrowserURL from '../../util/BrowserURL';
 
 const style = {
   wrapper: {
     width: '100%',
   },
   image: {
-    width: '30%'
-  }
+    width: '30%',
+  },
```

Comment:
is it a good practice to keep adding comma and ;  at the end of the line?

Reply:
These are options that can be configured. 

### trailingComma
If I add a new property at the end of an object the PR only shows the modified line. Without this option, it shows the modifier line and the line before with the addition of comma.

**Without trailing comma**
```diff
  width: '30%', // shows this line as modified as well
+ height: '100%''
```

**With trailling comma**
```diff
  width: '30%', // this line won't show as modified
+ height: '100%,''
```

### Semicolon
This is a good reference: https://javascript.info/structure#semicolon. Their recommendation:
> We recommend putting semicolons between statements even if they are separated by newlines. This rule is widely adopted by the community. Let’s note once again – it is possible to leave out semicolons most of the time. But it’s safer – especially for a beginner – to use them.

In my experience, I see people with very good JavaScript skills not using it. But projects more beginner-friendly like `create-react-app` using it.

Reply:
nice